### DISCUSSION TITLE
How to remove contributor of deleted commits?

### DISCUSSION THREAD

**Discussion by zacharyberson** [community/community#22357] [New to GitHub]
<p>Want to remove contributor from a project. The only commit made by the contributor has been deleted and thus the contributor doesn’t appear in the list when I click “manage access” under the project settings. But on the project main page it still shows the contributor. How to remove the contributor from the project?</p>

**Comment by jsoref** [comment]
<p>What do you mean the commit has been <code>deleted</code>?</p>
<p>If you <code>reverted</code> the commit, it’s still there and thus you haven’t <code>deleted</code> it.</p>
<p>If you’ve actually rewritten history so that the commit isn’t in the history for any branches or tags, then that’s a different story (I haven’t checked GitHub’s contributors algorithm in detail recently…)</p>

**Comment by zacharyberson** [comment]
<p>I used git reset --HARD~1 then git push --force origin main</p>

**Comment by jsoref** [comment, marked as answer]
<p>I’d probably contact<br>
<a href="https://support.github.com" class="onebox" target="_blank" rel="noopener nofollow ugc">https://support.github.com</a></p>
<p>But, if my repository didn’t have any forks, I’d make a full local clone, create a new empty github repository and then push my local clone into it. I could then validate the contributors don’t have the deleted entry, rename the old repo, and rename the replacement to fill it. (And finally delete the old one.)</p>

**Comment by zacharyberson** [comment]
<p>I contacted support and they removed the second contributor. Thanks.</p>
